apps/updownload/src/components/Import.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <Molgenis v-model="session">
+    taskList {{ taskList }}
```

Comment:
This is another feature I think? Maybe put it in a separate PR or at least in a separate commit? To keep the changelog accurate?

Reply:
ah, well spotted. This is an error should not have been committed. (is part of another feature under development)